configs/components/echam/echam.datasets.yaml

```diff
@@ -133,3 +133,12 @@ forcing_sources:
                          to: 2014
                  "${input_dir}/MAC-SP_${scenario}.nc":
                          from: 2015
+
+        # carbon emissions for echam6:with_lco2_emis
+        carbon_emissions_hist: "${forcing_dir}/${resolution}_carbon_emissions_ssp126.nc" # for years 1948 to 2014 all the same
+        carbon_emissions_ssp126: "${forcing_dir}/${resolution}_carbon_emissions_ssp126.nc"
+        carbon_emissions_ssp245: "${forcing_dir}/${resolution}_carbon_emissions_ssp245.nc"
+        carbon_emissions_ssp370: "${forcing_dir}/${resolution}_carbon_emissions_ssp370.nc"
+        carbon_emissions_ssp534os: "${forcing_dir}/${resolution}_carbon_emissions_ssp534.nc"
+        carbon_emissions_ssp585: "${forcing_dir}/${resolution}_carbon_emissions_ssp585.nc"
```

Comment:
We can avoid this repetition when `feat/ssp_generalization` is merged. I'll take care of that fix.

Reply:
This comment was ignored. Because we got no reply from the user, this PR was never merged.